float32 trackpad_x
float32 trackpad_y
float32 trigger
bool menu_button
bool trigger_button
bool trackpad_touch
bool trackpad_button
bool grip_button
int8 role
string time

# Pose data
geometry_msgs/TransformStamped abs_pose
geometry_msgs/TransformStamped rel_pose